src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssPropertiesAndValues = sourceString.split(';');
+  const result = {};
+
+  cssPropertiesAndValues.forEach((propertyValuePair) => {
+    const trimmedItem = propertyValuePair.trim();
+
+    if (trimmedItem.length < 4) {
```

Comment:
The check for 'trimmedItem.length < 4' seems arbitrary. It might exclude valid CSS declarations that have short keys and values, like 'a:b;'. A more reliable approach would be to check if both key and value exist after the split operation.